src/components/Header/header.jsx

```diff
@@ -6,22 +6,19 @@ import Logo from '../../Utilidades/ecommerceLogo.png';
 import './header.css'
 
 
-const Header = () => {
+const Header = (props) => {
```

Comment:
Trata igual de hacer el destructure de los props aquí

```suggestion
const Header = ({ handlerAllProducts }) => {
```
esto te va a permitir en tu linea 14 hacer solo esto 

```
<div className='header-logo' onClick={handlerAllProducts()}>
```

Reply:
También se me había pasado pero los imports trata de dejar con mayúscula aquellos que son componentes
```
import shoppingIcon from '../../Utilidades/Shopping.svg';
import hamburger from '../../Utilidades/hamburger.svg';
import logo from '../../Utilidades/ecommerceLogo.png';
```